app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1161,9 +1166,9 @@ class PipelineSampleReport extends React.Component {
     // expand all real genera
     this.expandAll = true;
     this.expandedGenera = [];
-    $(".report-row-species.real-genus").removeClass("hidden");
-    $(".report-arrow-down.real-genus").removeClass("hidden");
-    $(".report-arrow-right.real-genus").addClass("hidden");
+    $(".report-row-species").removeClass("hidden");
+    $(".report-arrow-down").removeClass("hidden");
+    $(".report-arrow-right").addClass("hidden");
```

Comment:
why this change?

Reply:
For #1236 since the ".real-genus" prevented it from opening the 'fake genus' lines